x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/use_sub_action_mutation.tsx

```diff
@@ -20,7 +20,6 @@ export const useSubActionMutation = <P, R>({
   connectorId,
   subAction,
   subActionParams,
-  disabled = false,
```

Comment:
is there a point in keeping the `disabled` optional property in the interface a few lines above? It doesn't seem to be used anywhere